InfoMovies/InfoMovies/Views/BasePage.xaml.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace InfoMovies.Views
+{
+	[XamlCompilation(XamlCompilationOptions.Compile)]
```

Comment:
Why is this line needed?

Reply:
to check Xaml errors at compile time and improve application load.

Reply:
Sure, but you’ve added it at application  level. There’s no need to add it again in every page.

Reply:
Really? I didn't know!
Thanks for that!